boot/zephyr/main.c

```diff
@@ -223,6 +227,36 @@ void main(void)
     }
 #endif
 
+#ifdef CONFIG_BOOT_SERIAL_UART_DETECT
+   
+    char str[2];
+    volatile int newline = 0;
+    volatile int timeout = 10000;
+    int step = 1;
+   
+    BOOT_LOG_INF("Wait:");
+    while(1){
+        console_read(str, 2, &newline);
+        if(newline == 1){
```

Comment:
And this `if (newline == 1) {`, with the space after the `if` and before the brace. There are a few similar below.